DevCycle.SDK.Server.Cloud.MSTests/TestResponse.cs

```diff
@@ -30,7 +30,7 @@ public static Variable<bool> GetVariableByKeyAsync()
             {
                 {
                     "test-false",
-                    new Variable<bool>(Guid.NewGuid().ToString(), "test-false", TypeEnum.Boolean, false)
```

Comment:
why remove the random id?

Reply:
its not part of the variable class as defined in Node (which is our reference)